src/Client/Components/Pages/DynamicLogging/Index.razor

```diff
@@ -1,38 +1,27 @@
-﻿@page "/dynamic-logging"
+@page "/logging/get-started"
+@page "/dynamic-logging"
 @page "/steeltoe-logging"
 
-<Template Name="Dynamic Logging" Features="@features" DocsURL="/docs/logging">
-	<Description>XXXXXXX</Description>
-	<WhatIs>XXXXXXX</WhatIs>
-	<WhyUse>XXXXXXX</WhyUse>
-	<FeatureTemplate Context="Feature">
-		<div class="row component-feature no-gutters">
-			<div class="col-6 text-center"><div class="component-feature-img"><img class="img-fluid" src="@Feature.Img" /></div></div>
-			<div class="col">
-				<div class="component-feature-title">@Feature.Title</div>
-				<div class="component-feature-desc">@Feature.Description</div>
-			</div>
-		</div>
-	</FeatureTemplate>
-	<AltFeatureTemplate Context="Feature">
-		<div class="row component-feature no-gutters">
-			<div class="col-6">
-				<div class="component-feature-title">@Feature.Title</div>
-				<div class="component-feature-desc">@Feature.Description</div>
-			</div>
-			<div class="col-6 text-center"><div class="component-feature-img"><img class="img-fluid" src="@Feature.Img" /></div></div>
-		</div>
-	</AltFeatureTemplate>
-</Template>
+@inject IHttpContextAccessor HttpContextAccessor
+@inject IOptionsSnapshot<DocsSiteOptions> DocsSiteOptions
+<Breadcrumbs NavName="Getting Started"
+						 ParentLinkName="Logging"
+						 ParentLinkUrl="/logging" />
+<GetStarted>
+    <ComponentName>Logging</ComponentName>
+    <Tutorials>
+        <h1>This page has moved, find the new location <NavLink href="@(DocsSiteOptions.Value.BaseAddress + "/guides/")">here</NavLink></h1>
```

Comment:
As in what links to this page? Hopefully nothing since there was never actually anything here. Redirecting over to /guides seemed like the closest we could get to the original intent, though we would still need to add a guide. Would you prefer linking to the docs?

Reply:
Yes. Closing, as the change has been applied already.